.github/workflows/nightly.yaml

```diff
@@ -75,6 +70,11 @@ jobs:
         with:
           version: v3.7.1-2
 
+      - name: Set up for matrix
```

Comment:
Are we checking if changes occurred, or is nightly just checking current main?

Reply:
Nightly is checking the current main.